src/SIPdump.h

```diff
@@ -361,8 +361,9 @@ int find_value(const char *value, const char *buffer, char *outbuf,
 		}
 	}
 
-	strncpy(outbuf, tempbuf, outbuf_len);
-	outbuf[outbuf_len] = 0;
+	strncpy(outbuf, tempbuf, outbuf_len - 1);
+	if (outbuf_len > 0)
+		outbuf[outbuf_len - 1] = 0;
```

Comment:
Good question!

I don't think that `outbuf_len` can be zero at this point. I copy-pasted this `if (outbuf_len > 0)` check from the strncpy's man page.

`man strncpy` has,

```
strncpy(buf, str, buflen - 1);
if (buflen > 0)
    buf[buflen - 1]= '\0';
```

This example has the same problem!

I will review, and update this patch soon. Thanks!